ensembl_dbcopy/admin.py

```diff
@@ -51,10 +69,15 @@ class Media:
         }
 
     # form = HostRecordForm
-    inlines = (GroupInline,)
-    list_display = ('name', 'port', 'mysql_user', 'virtual_machine', 'mysqld_file_owner')
-    fields = ('name', 'port', 'mysql_user', 'virtual_machine', 'mysqld_file_owner')
-    search_fields = ('name', 'port', 'mysql_user', 'virtual_machine', 'mysqld_file_owner')
+    #inlines = (GroupInline,)
+    list_display = ('name', 'port', 'mysql_user', 'virtual_machine', 'mysqld_file_owner', 'get_groups' ,'active')
+    fields = ('name', 'port', 'mysql_user', 'virtual_machine', 'mysqld_file_owner','groups', 'active')
+    search_fields = ('name', 'port', 'mysql_user', 'virtual_machine', 'mysqld_file_owner', 'active')
+    
+    def get_groups(self, obj):
+        return ", ".join([str(g) for g in obj.groups.all()])
+
+    get_groups.short_description = 'Groups '
 
 
 class OverallStatusFilter(SimpleListFilter):
```

Comment:
And this one in a file names `filters.py` :-)

Reply:
sure i will do that